docker-compose.yml

```diff
@@ -1,11 +1,37 @@
-app:
-  mem_limit: 300m
-  memswap_limit: 1g
-  env_file:
-    .env
-  build: .
-  ports:
-    - '3000:8000'
-  volumes:
-    - .:/home/nodejs/app
-    - /home/nodejs/app/node_modules
+version: '2'
+
+services:
+  mongodb:
+    container_name: ya-mongo-db
+    image: alexpunct/mongo:3.4
```

Comment:
You probably can even higher to 3.6

Reply:
@markelog in general i want to use official image later on, but for now its ok